src/atoms/price/Price.stories.js

```diff
@@ -2,77 +2,30 @@ import { storiesOf } from '@storybook/vue'
 
 import APrice from './Price.vue'
 
-// @vue/component
-const defaultData = {
-  components: { APrice },
-  data: () => {
-    return {
-      price: '$1 400.00',
-      special: '$1 299 00',
-      old: '$1 999.99'
-    }
-  }
-}
-
 storiesOf('Atoms/Price', module)
   .addParameters({ info: true })
-  // @vue/component
   .add('Default', () => ({
-    ...defaultData,
-    template: `
-      <a-price>
-        {{ price }}
-      </a-price>
-    `
-  })
-  )
-  // @vue/component
-  .add('Special Price', () => ({
-    ...defaultData,
-    template: `
-      <a-price
-        :special-price="special"
-        aria-label-special="Special price:"
-      />
-    `
+    components: { APrice },
+    template: `<a-price regular-price="$36,00" />`
```

Comment:
Would be great to have some knobs (even if they are not super useful) to use in this story for some values :slightly_smiling_face:

Reply:
knobs added